tardis/montecarlo/packet_source.py

```diff
@@ -76,8 +76,7 @@ def create_blackbody_packet_nus(T, no_of_packets, rng, l_samples=1000):
 
         Parameters
         ----------
-        T : float
-            temperature
+        temperature : float
```

Comment:
```suggestion
        temperature : float
            Temperature.
```
It would be great if you could research and add a better description of the variable.